viewing_party/party.py

```diff
@@ -0,0 +1,151 @@
+
+#Wave 1
+
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating:
+        movie_dict = {
+        "title" : movie_title,
+        "genre" : genre,
+        "rating" : rating}
+        return movie_dict
+    else:
+        return None
+
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"]
+    for movie in watchlist:
+        if movie["title"] == title:
+            title = movie
+    if title in user_data["watchlist"]:
```

Comment:
Note that this second conditional test is not necessary because you've already determined that there is a movie with the specified title. Consider how you can refactor to remove this extra test.